v5/releases/cli3.1.4.rst

```diff
@@ -0,0 +1,47 @@
+======================
+PROS CLI 3.1.4 Release
+======================
+
+.. post:: 12 February, 2019
+   :tags: blog, cli-release
+
+- Added support for supplying upload arguments to ``mu``/``mut``/``ut``
+- Add V5 screen capture with ``prosv5 v5 capture``
+- New UI experience for creating, upgrading, and uploading projects
+- Add binary compression to save space on the V5
+- Add support for hot/cold linking
+- Add wireless downloading support
+- Bug fixes to V5 upload protocol
+
+Updating to the new version
+===========================
+
+Python PIP
+----------
+
+Run pip install https://github.com/purduesigbots/pros-cli3/releases/download/3.1.2/pros_cli_v5-3.1.2-py3-none-any.whl (or pip3, depending on your system)
+
+Windows
+-------
+
+Download and run the appropriate installer
+
+macOS
+-----
+
+See below for the new recommended method of installing the CLI
+
+macOS installs
+~~~~~~~~~~~~~~
+
+Install Homebrew if it's not already installed. This may take a while.
+Run brew tap purduesigbots/pros
+Run brew cask install gcc-arm-embedded pros-editor (if you want to install only the CLI, you can run brew cask install gcc-arm-embedded && brew install pros-cli instead)
```

Comment:
need to update this. Homebrew Cask recently removed gcc-arm-embedded, so my recommended way to install the toolchain for now is to run `brew tap osx-cross/arm && brew install gcc-arm-bin` (see [relevant docs page](https://pros.cs.purdue.edu/v5/getting-started/macos.html))

Reply:
```suggestion
Run ``brew install gcc-arm-bin && brew cask install pros-editor`` (if you want to install only the CLI, you can run ``brew install gcc-arm-bin pros-cli`` instead)